juniper_official/Protocols/Lldp/lldp-session-statistics.rule

```diff
@@ -118,20 +121,23 @@ healthbot {
                 sensor lldp-sensor {
                     where "/lldp/interfaces/interface/@name =~ /{{if-name}}/";
                     path "/lldp/interfaces/interface/@name";
+                    zero-suppression;
```

Comment:
Same here. Should we be zero-suppressing type string?

Reply:
Corrected